src/storage/exec/QueryUtils.h

```diff
@@ -73,6 +73,9 @@ class QueryUtils final {
 
       if (nullType == NullType::UNKNOWN_PROP) {
         VLOG(1) << "Fail to read prop " << propName;
+        if (!field) {
```

Comment:
Why not just return null outside the if block？

Reply:
It is null here~ NullType::UNKNOWN_PROP

Reply:
IMO, NullType::UNKNOWN_PROP is not necessary, which can lead to more problems due to graphd's unreasonable handling of runtime errors.
For example, filter operator execution will throw a runtime error if UNKNOWN_PROP is fed to it. 
https://github.com/vesoft-inc/nebula/blob/7f185c969f33ac7bc85a57de0ec3d7620253c24b/src/graph/executor/query/FilterExecutor.cpp#L67-L69